stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -174,6 +184,26 @@ public fun DefaultMessageItem(
     }
 }
 
+/**
+ * A system message is a message generated by a system event, such as updating the channel or muting a user.
+ *
+ * @param systemMessageItem The system message item to show.
+ * @param modifier Modifier for styling.
+ */
+@Composable
+public fun SystemMessage(
+    systemMessageItem: SystemMessageItem,
+    modifier: Modifier = Modifier,
+) {
+    Text(
+        modifier = modifier.padding(top = 12.dp, start = 16.dp, end = 16.dp, bottom = 12.dp),
```

Comment:
How about using the `vertical` and `horizontal` parameters instead like the below?

```kotlin
.padding(vertical = 12.dp, horizontal = 16.dp)
```